main/zoe/api/zoe.md

```diff
@@ -2,20 +2,34 @@
 
 <Zoe-Version/>
 
-Zoe is a long-lived and well-trusted contract that enforces offer safety for the contracts that use it. Zoe has a single `inviteIssuer` for the entirety of its lifetime. By having a reference to Zoe, a user can get the `inviteIssuer` and thus validate any `invite` they receive from someone else.
-
-## zoe.getInviteIssuer()
+Zoe provides a framework for deploying and working with smart contracts. It is accessed 
+as a long-lived and well-trusted service that enforces offer safety for the contracts that use it. Zoe has a single `inviteIssuer` for the entirety of its lifetime. By having a reference to Zoe, a user can get the `inviteIssuer` and thus validate any `invite` they receive from someone else.
+
+::: tip Zoe is accessed asynchronously
+The Zoe service is accessed asynchronously, using a standards-track library extension
+to JavaScript that uses promises as remote references. In code, the Zoe service instance 
+is refered to via `zoe`, whcih only supports asynchronous invocation. Operations are 
```

Comment:
I think there's typos in this document. Can we run a spell checker on it?

Reply:
done